MathPortal/MathPortal/Source/UserInterface/TabBarViewControllers/BrowseTasks/TaskDetails/TaskDetailsViewControllerSolutionCell.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  TaskDetailsViewControllerSolutionCell.swift
+//  MathPortal
+//
+//  Created by Petra Čačkov on 30/09/2019.
+//  Copyright © 2019 Petra Čačkov. All rights reserved.
+//
+
+import UIKit
+
+class TaskDetailsViewControllerSolutionCell: UITableViewCell {
```

Comment:
Redo.
- Adding views for every setupCell will keep piling up views
- Inconsistency with asynchronousy for fetching user
- Should not save parameters for TaskSolution nor User
- setupEquationView should be private